app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/showscreen/ShowScreen.kt

```diff
@@ -0,0 +1,671 @@
+                        .aspectRatio(1f)
+                )
+                ShowScreenMediaBasicInfo(
+                    title = mockMovie.title,
+                    date = mockMovie.date,
+                    language = mockMovie.language,
+                    progress = mockMovie.rating,
+                    duration = getMinutesString(mockMovie.duration),
+                    seasons = null,
+                    modifier = Modifier
+                        .align(alignment = Alignment.BottomCenter)
+                        .fillMaxWidth()
+                        .padding(horizontal = dimensionResource(R.dimen.show_screen_horizontal_padding))
+
+                )
+            }
+            Spacer(
+                modifier = Modifier.height(dimensionResource(R.dimen.show_screen_spacer_height_1))
+            )
+            HorizontalDivider(
+                thickness = dimensionResource(R.dimen.horizontal_divider_thickness),
+                color = MaterialTheme.colorScheme.scrim,
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(horizontal = dimensionResource(R.dimen.show_screen_horizontal_padding))
+            )
+            Spacer(
+                modifier = Modifier.height(dimensionResource(R.dimen.show_screen_spacer_height_1))
+            )
+            Text(
+                text = mockMovie.overview,
+                style = MaterialTheme.typography.bodySmall.copy(
+                    fontWeight = FontWeight.Normal,
+                    fontSize = 12.sp,
+                    color = white_CC,
+                    lineHeight = 16.sp
+                ),
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(horizontal = dimensionResource(R.dimen.show_screen_horizontal_padding))
+            )
+            Spacer(
+                modifier = Modifier.height(dimensionResource(R.dimen.show_screen_spacer_height_2))
+            )
+            ExpandedButton(
+                onClick = {
+                    playTrailer(mockMovie.trailerUrl)
+                },
+                modifier = Modifier.padding(horizontal = dimensionResource(R.dimen.show_screen_horizontal_padding)),
+                content = {
+                    Row {
+                        Icon(
+                            imageVector = Icons.Outlined.PlayCircle,
+                            contentDescription = null
+                        )
+                        Spacer(
+                            modifier = Modifier.width(dimensionResource(R.dimen.play_trailer_items_padding))
+                        )
+                        Text(
+                            text = stringResource(R.string.play_trailer)
+                        )
+                    }
+                }
+            )
+            Spacer(
+                modifier = Modifier.height(dimensionResource(R.dimen.show_screen_spacer_height_3))
+            )
+            Text(
```

Comment:
This text can be added to the `MainActors` composable

Reply:
done